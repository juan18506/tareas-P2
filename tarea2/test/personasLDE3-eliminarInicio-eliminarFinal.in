crearListaPersonas

crearAgenda
crearPersona 1 11 Persona1
crearFecha 1/11/11
crearEvento 1 Descripcion1
agregarEventoPersona
insertarPersonaListaPersonas 1

crearAgenda
crearPersona 2 22 Persona2
crearFecha 2/22/22
crearEvento 2 Descripcion2
agregarEventoPersona
insertarPersonaListaPersonas 2

crearAgenda
crearPersona 3 33 Persona3
crearFecha 3/33/33
crearEvento 3 Descripcion3
agregarEventoPersona
insertarPersonaListaPersonas 2

# debe haber 3 personas en la lista
imprimirCantidadListaPersonas

# la persona 3 debe estar en la posicion 2
imprimirListaPersonas

# elimino el inicio, es decir la persona 1
eliminarInicioListaPersonas

# lista sin la persona 1
imprimirListaPersonas

# elimino el final, es decir la persona 2
eliminarFinalListaPersonas

# debe quedar solo 1 persona en la lista
imprimirCantidadListaPersonas

# debe quedar solo la persona 3 en la lista
imprimirListaPersonas

# elimino el inicio, la lista queda vacia
eliminarInicioListaPersonas

# no se debe imprimir nada
imprimirListaPersonas

crearAgenda
crearPersona 4 44 Persona4
crearFecha 4/44/44
crearEvento 4 Descripcion4
agregarEventoPersona
insertarPersonaListaPersonas 2

# ahora debe haber solo 1 persona en la lista
imprimirListaPersonas

liberarListaPersonas

Fin