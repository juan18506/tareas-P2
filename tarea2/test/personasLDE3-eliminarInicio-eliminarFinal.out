1>2>3>4>5>6>7>8>9>10>11>12>13>14>15>16>17>18>19>20>#  debe haber 3 personas en la lista.
21>La cantidad de personas en la lista es: 3.
22>#  la persona 3 debe estar en la posicion 2.
23>Persona 1:  Persona1, 11 años
Evento 1:  Descripcion1
Fecha: 1/11/11
Persona 3:  Persona3, 33 años
Evento 3:  Descripcion3
Fecha: 3/33/33
Persona 2:  Persona2, 22 años
Evento 2:  Descripcion2
Fecha: 2/22/22
24>#  elimino el inicio, es decir la persona 1.
25>26>#  lista sin la persona 1.
27>Persona 3:  Persona3, 33 años
Evento 3:  Descripcion3
Fecha: 3/33/33
Persona 2:  Persona2, 22 años
Evento 2:  Descripcion2
Fecha: 2/22/22
28>#  elimino el final, es decir la persona 2.
29>30>#  debe quedar solo 1 persona en la lista.
31>La cantidad de personas en la lista es: 1.
32>#  debe quedar solo la persona 3 en la lista.
33>Persona 3:  Persona3, 33 años
Evento 3:  Descripcion3
Fecha: 3/33/33
34>#  elimino el inicio, la lista queda vacia.
35>36>#  no se debe imprimir nada.
37>38>39>40>41>42>43>44>#  ahora debe haber solo 1 persona en la lista.
45>Persona 4:  Persona4, 44 años
Evento 4:  Descripcion4
Fecha: 4/44/44
46>47>Fin.
